types/FrameTypes.ts

```diff
@@ -28,12 +28,23 @@ export type FrameType = {
 };
 
 export type Frame = TextFrame | ImageFrame;
+export type ImageVariableSource = {
```

Comment:
ImageUrlVariableSource

Reply:
Isn't it confusing to add Url in the name? I mean it doesn't rely on any url after all

Reply:
Yea, I get it, but the `sourceType` for that type is `urlVariable` so it would make sense for now to use `ImageUrlVariableSoruce`.

Reply:
Ok, agreed